host_reference/contract_reference.h

```diff
@@ -1,9 +1,9 @@
 #pragma once
 
-#include <blas_reference.h>
+#include <host_utils.h>
+//#include <blas_reference.h>
```

Comment:
If it's not being used just delete rather than comment out.

Reply:
9accde037